cicecore/drivers/nuopc/cmeps/ice_import_export.F90

```diff
@@ -481,117 +502,161 @@ subroutine ice_import( importState, rc )
+    !
+    !                   +---------------------------+
+    !                   |       Ocean Import        |
+    !                   +---------------------------+
+
+    ! Sea surface temperature --------------------------------------------
     call state_getimport(importState, 'So_t', output=aflds, index=1, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
+    ! Sea surface salinity ------------------------------------------------
     call state_getimport(importState, 'So_s', output=aflds, index=2, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
-    ! import atm states
+    ! Deep ocean heat flux at base of mixed layer -------------------------
 
+    !call state_getimport(importState, 'Fioo_q', output=aflds, index=9, &
+    !     areacor=med2mod_areacor, rc=rc)
+    !if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    !call ESMF_LogWrite("Ocean/ice flux imported! ", ESMF_LOGMSG_INFO)
+    ! import atm fluxes
+
+        
+    !                   +---------------------------+
+    !                   |       Atmos Import        |
+    !                   +---------------------------+
+    
+    ! Lowest atmospheric height -------------------------------------------
     call state_getimport(importState, 'Sa_z', output=aflds, index=3, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
+    ! Calc/Imp surface pressure -------------------------------------------------------------
     if (State_FldChk(importState, 'Sa_ptem') .and. State_fldchk(importState, 'Sa_dens')) then
        call state_getimport(importState, 'Sa_ptem', output=aflds, index=4, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
        call state_getimport(importState, 'Sa_dens', output=aflds, index=5, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
-    else if (State_FldChk(importState, 'Sa_pbot')) then
-       call state_getimport(importState, 'Sa_pbot', output=aflds, index=6, rc=rc)
+    else if (State_FldChk(importState, 'Sa_pslv')) then
+       call state_getimport(importState, 'Sa_pslv', output=aflds, index=6, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
-    else
-       call abort_ice(trim(subname)//&
-            ": ERROR either Sa_ptem and Sa_dens OR Sa_pbot must be in import state")
-    end if
+    else 
+       ! Nothing is defined so using a fill value of 100000Pa
+       
+       !call state_getimport(importState, 'Sa_tbot', output=aflds, index=6, rc=rc)
+       
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+         do iblk = 1, nblocks
+            do j = 1,ny_block 
+               do i = 1,nx_block 
+               aflds(i,j,6,iblk) = 100000._ESMF_KIND_R8 !
+               enddo
+            enddo
+         enddo
+    endif
 
-    call state_getimport(importState, 'Sa_tbot', output=aflds, index=7, rc=rc)
-    if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
-    call state_getimport(importState, 'Sa_shum', output=aflds, index=8, rc=rc)
+    ! Surface temperature ---------------------------------------------------
```

Comment:
@SmithJos13 Again, please do not remove/change any exiting variable. It will have downstream effect in other cases.